cypress/support/commands.ts

```diff
@@ -218,8 +219,15 @@ Cypress.Commands.add('draganddrop', (draggableSelector, droppableSelector, optio
 			const pageY = coords.top + coords.height / 2;
 
 			if (draggableSelector) {
-				// We can't use realMouseDown here because it hangs headless run
-				cy.get(draggableSelector).trigger('mousedown');
+				cy.ifCanvasVersion(
+					() => {
+						// We can't use realMouseDown here because it hangs headless run
```

Comment:
Seems like this comment is no longer relevant. At least locally realMouseDown no longer hangs the tests. So perhaps we could simplify this by always using it?